assets/js/validation.js

```diff
@@ -1,21 +1,32 @@
 const axios = require('axios');
 
-let name = document.getElementById('name');
-let validationResult = document.getElementById('validation-result');
-const validateName = function () {
-    validationResult.innerText = '...';
-    axios.post(validationResult.dataset.path, {input: name.value})
-        .then(function(response) {
-            if (response.data.valid) {
-                validationResult.innerHTML = ":)";
-            } else {
-                validationResult.innerHTML = ":(";
-            }
+const name = document.getElementById('name');
+const nameValidation = document.getElementById('validation-result-name');
+
+const team = document.getElementById('team');
+const teamValidation = document.getElementById('validation-result-team');
+
+const validate = (input, validationText) => {
+    validationText.innerText = '...';
+    axios
+        .post(validationText.dataset.path, { input: input.value })
+        .then(response => {
+            response.data.valid
+            ? (validationText.innerHTML = ':)')
+            : (validationText.innerHTML = ':(');
         })
-        .catch(function (error) {
-            validationResult.innerText = 'Error: ' + error;
+        .catch(error => {
```

Comment:
:+1: Kad naudoiji nauajusias JavaScript galimybes.

Senos naršyklės dar gali nepalaikyti _arrow notation_.
Tai reikia sužiūrėti, ar pagal _webpack_ nustatymus užtikrinama senų naršyklių palaikymą.

Reply:
Kadangi ant mūsų akademijos starter projektų sukasi susetup'intas babel'is tai nesukau galvos ir panaudojau ES6 :)